test/lib/validators/content_view_puppet_module_validator_test.rb

```diff
@@ -25,7 +25,8 @@ def setup
     end
 
     test "passes if name and author provided" do
-      PuppetModule.create!(:name => "module name", :author => "module author", :uuid => '9932943299423', :repositories => [@repository])
+      puppet = PuppetModule.create!(:name => "module name", :author => "module author", :uuid => '9932943299423')
+      puppet.repositories << @repository
```

Comment:
No, not with a Hash. You have to either:

1. Set the values explicitly
2. Use a ActionController::Parameters object and call permit on the keys you want to permit (see http://api.rubyonrails.org/v4.2.4/classes/ActionController/Parameters.html#method-i-permit)
3. Use a factory

I'm a bit surprised the previous line doesn't throw an error though. I'll try to dig into it tomorrow.